public/coffee/ide/pdfng/directives/pdfRenderer.coffee

```diff
@@ -57,15 +62,20 @@ define [
 					pdfDocument.getDestinations()
 				return @destinations.then (all) ->
 					all[dest]
+				, (error) ->
+					console.log 'ERROR', error
 
-				@document.then (pdfDocument) ->
-					pdfDocument.getDestination(dest)
-
+				# @document.then (pdfDocument) ->
+				# 	pdfDocument.getDestination(dest)
+				# , (error) ->
+				# 	console.log 'ERROR', error
```

Comment:
The commented code will be what we use when we upgrade to a newer version of pdfjs with a direct method for getDestination, I'll change the comment to make that clearer.